.goreleaser.yml

```diff
@@ -1,11 +1,16 @@
+version: 2
 builds:
 - env:
   - CGO_ENABLED=0
   goarch:
   - amd64
+  - arm64
   goos:
   - linux
   - darwin
+  ignore:
+  - goos: linux
+    goarch: arm64
 archives:
 - name_template: "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}"
```

Comment:
@blgm,  `name_template` needs to be changed
I realized this after merging the pull request. I am sorry